raddb/mods-available/imap

```diff
@@ -0,0 +1,159 @@
+		#
+		#  Specifies the certificate that directly signed the certificate presented by the
+		#  HTTPS server.
+		#
+		#  This configuration option can be used to prevent certificates passing validation that
+		#  were signed by other intermediary CAs or root CAs in the trusted certificate chain.
+		#
+#		ca_issuer_file     = "${certdir}/caissuer.pem"
+
+		#
+		#  ca_path:: A directory containing multiple root CA certs named by their hash
+		#
+		#  See the OpenSSL documentation for more details:
+		#  - https://www.openssl.org/docs/man1.1.0/man3/SSL_CTX_set_default_verify_paths.html
+		#  - https://www.openssl.org/docs/man1.1.1/man1/c_rehash.html
+		#
+		#  This configuration option should only be used when the HTTPS server being contacted
+		#  is not known ahead of time (using a URL from an external source), and/or the CA used
+		#  to sign the HTTPS server certificate is unknown.
+		#
+#		ca_path	             = "${certdir}"
+
+		#
+		#  certificate_file:: PEM formatted file containing the certificate we present to the HTTPS server
+		#
+		#  Specifies a certificate and any intermediary CAs we should send to the HTTPS server.
+		#
+		#  This file should usually contain the client certificate file first, then any
+		#  intermediary signing CAs, shallowest (direct signee of the certificate_file)
+		#  to deepest (signed directly by the root CA).
+		#
+#		certificate_file     = /path/to/radius.pem
+
+		#
+		#  private_key_file:: PEM formatted file containing the private key for the specified certificate_file
+		#
+		#  Must be specified if certificate_file is being used.
+		#
+#		private_key_file     = /path/to/radius.key
+
+		#
+		#  private_key_password:: Password used to decrypt the private key file.
+		#
+		#  Should only be specified in the private_key_file is encrypted.
+		#
+#		private_key_password = "supersecret"
+
+		#
+		#  random_file:: Source of random data used for various cryptographic functions.
+		#
+#		random_file          = /dev/urandom
+
+		#
+		#  check_cert:: Server certificate verification requirements.
+		#
+		#  May be one of:
+		#
+		#  [options="header,autowidth"]
+		#  |===
+		#  | Option | Description
+		#  | `no`   | Server certificate can be signed by any CA or be self-signed.
+		#  | `yes`  | Server certificate must be issued by one of the trusted CAs.
+		#  |===
+		#
+		#  Default is `yes`
+		#
+#		check_cert = no
+
+		#
+		# curl_ssl_opt:: To what degree curl should use ssl
+		#
+		# May be one of:
+		#
+		#  [options="header,autowidth"]
+                #  |===
+                #  | Option | Description
+                #  | `all`   | Require ssl validation to accept login
+                #  | `none`  | Do not try to establish ssl connection
+                #  | `try`  | Try to establish ssl connection, continue even if it cannot
+                #  |===
+                #
+                #  default is 'try'
+                #
+#		curl_ssl_opt = try
```

Comment:
Should be `require_cert`, with the same value names as the LDAP module.

https://github.com/FreeRADIUS/freeradius-server/blob/de1f57c8922d5b870c23ee715c39bdaea36a4ade/src/lib/ldap/base.c#L79.

https://github.com/FreeRADIUS/freeradius-server/blob/master/raddb/mods-available/ldap#L735